scripts/jenkins/groovy/hadoopCommands.groovy

```diff
@@ -54,7 +54,7 @@ private GString getCommandHadoop(final stageConfig) {
 private GString getCommandStandalone(final stageConfig) {
     def defaultPort = 54321
     return """
-            java -cp build/h2o.jar:\$(cat /opt/hive-jdbc-cp) water.H2OApp \\
+            java -cp h2o-hadoop-*/h2o-${stageConfig.customData.distribution}${stageConfig.customData.version}-assembly/build/libs/h2odriver.jar:\$(cat /opt/hive-jdbc-cp) water.H2OApp \\
```

Comment:
The intention here is to use the regular h2o.jar to connect to Hadoop. We are testing standalone h2o.jar here.

Reply:
I thought it is recommended to use h2odriver.jar even when not running on hadoop but interacting with HDFS so that matching hadoop libs are used

Reply:
The original purpose of the test was that the built-in Hadoop classes can be used in most cases (!) to connect to Hadoop as well.

For more complex cases it is absolutely recommended what you are saying. But here we want to test the h2o.jar.

Reply:
than maybe we should add another stage to test that h2odriver.jar works in standalone as well, wdyt?

Reply:
That makes sense, that case is not covered currently.

Reply:
Will do.

Reply:
@honzasterba did this. Done. Thank you.